rdt-open-reader/src/main/java/io/ona/rdt/util/Utils.java

```diff
@@ -225,4 +228,48 @@ public static boolean isValidJSONObject(String str) {
             return false;
         }
     }
+
+    public static void verifyUserAuthorization() {
+        UserVerifyAuthTask.getInstance().run();
+    }
+
+    private static class UserVerifyAuthTask {
+
+        private static UserVerifyAuthTask INSTANCE;
+        private boolean isFinish = true;
```

Comment:
Remove this variable

Reply:
used isFinish variable to maintain a call only one at a time

Reply:
now used AsyncTask API to get the task status